starknet_py/net/client.py

```diff
@@ -197,6 +197,24 @@ async def estimate_fee(
         :return: Estimated amount of Wei executing specified transaction will cost
         """
 
+    @abstractmethod
+    async def estimate_fee_bulk(
+        self,
+        txs: List[Union[InvokeFunction, Declare, DeployAccount]],
+        block_hash: Optional[Union[Hash, Tag]] = None,
+        block_number: Optional[Union[int, Tag]] = None,
+    ) -> List[EstimatedFee]:
+        """
+        Estimate how much Wei it will cost to run provided InvokeFunction
+
+        :param txs: Transactions to estimate
+        :param block_hash: Get code at specific block hash or
+                           at the block indicated by the literals `"pending"` or `"latest"`
+        :param block_number: Get code at given block number or at
+                             the block indicated by the literals `"pending"` or `"latest"`
+        :return: List of estimated amount of Wei executing specified transactions will cost
+        """
```

Comment:
I'm not sure if this will ever be implemented in the RPC. I'd prefer we only added this method to GatewayClient and not the interface.